packages/docusaurus-plugin-content-pages/src/index.ts

```diff
@@ -221,6 +221,7 @@ export default function pluginContentPages(
                         `${docuHash(aliasedSource)}.json`,
                       );
                     },
+                    mermaid: siteConfig.markdown?.mermaid,
```

Comment:
Let's pass the entire `siteConfig.markdown` in. This field is meant to be read by `mdx-loader`.

Reply:
Done